tox.ini

```diff
@@ -31,8 +31,6 @@ description = Integration tests; execution of interface tests from a charm's poi
 deps =
     .[unit_tests]
     jsonschema
-    # todo replace with pypi package when available
-    git+https://github.com/canonical/interface-tester-pytest
```

Comment:
shouldn't it be also removed from the `[testenv:unit]` part above in the file?

Reply:
very much so. Fixing this and another stray line.